test/Microsoft.ML.Tests/AnomalyDetectionTests.cs

```diff
@@ -30,7 +30,7 @@ public void RandomizedPcaTrainerBaselineTest()
             var transformedData = DetectAnomalyInMnistOneClass(trainPath, testPath);
 
             // Evaluate
-            var metrics = ML.AnomalyDetection.Evaluate(transformedData, k: 5);
+            var metrics = ML.AnomalyDetection.Evaluate(transformedData, falsePositiveCount: 5);
```

Comment:
>falsePositiveCount [](start = 72, length = 18)

`AnomalyDetectionMetrics` has `DetectionRateAtKFalsePositives`
If you remove `K` from here, I think you need to remove `K` from metric class as well.

Reply:
Let's do `DetectionRateAtFalsePositiveCount`.

---
In reply to: [267591549](https://github.com/dotnet/machinelearning/pull/3030#discussion_r267591549) [](ancestors = 267591549)